cmd/virt-controller/BUILD.bazel

```diff
@@ -10,6 +10,7 @@ go_library(
         "//pkg/monitoring/reflector/prometheus:go_default_library",
         "//pkg/monitoring/workqueue/prometheus:go_default_library",
         "//pkg/virt-controller/watch:go_default_library",
+        "//pkg/virtblock:go_default_library",
```

Comment:
Here, and everywhere else, it should be `virtblocks` rather than `virtblock`.

Reply:
Currently it just som pkg directory that I created under the kubevirt `pkg/`, I will rename it once will figure out how to make this work:)

Reply:
If you give it the right name from the start you're not going to need as much search and replace later on, but either one works as far as I'm concerned :)